app/helpers/merge_requests_helper.rb

```diff
@@ -59,7 +59,7 @@ def merge_request_back_link(merge_request, page, referrer)
 
     case page
     when "absorbing_organisation"
-      organisations_path(anchor: "merge-requests")
+      organisations_path(tab: "merge-requests")
```

Comment:
All places where previously we'd access the right tab on the organisations index with the anchor must now use the tab parameter.

The _Merge requests_ tab is `tab: "merge-requests"`, the _All organisations_ tab is `tab: "all-organisations"` (not currently used anywhere).

Reply:
I've changed this to now accept an anchor parameter being set within the route helper. 
For example, `organisations_path(anchor: "merge-requests")` will be treated the same `organisations_path(tab: "merge-requests")`. The anchor will still be removed and replaced with a tab parameter in the url. Why change this? In the absence of the parameter tab, if anchor is set instead of tab, then we'll use that value to determine which tab to show.

Two possible use cases:

- You want to swap in this data controller with another tab element without making changes in all places the path is used.
- A developer makes a mistake and uses the anchor key word instead of tab.